book/listings/css/1/main.rs

```diff
@@ -0,0 +1,52 @@
+use gdk::Display;
+use gtk::gdk;
+use gtk::prelude::*;
+use gtk::{Application, ApplicationWindow, Button, CssProvider, StyleContext};
+
+// ANCHOR: main
+fn main() {
+    // Create a new application
+    let app = Application::builder()
+        .application_id("org.gtk-rs.example")
+        .build();
+
+    // Connect to signals
+    app.connect_startup(|_| load_css());
+    app.connect_activate(build_ui);
+
+    // Run the application
+    app.run();
+}
+
+fn load_css() {
+    // Load the css file and add it to the provider
+    let provider = CssProvider::new();
+    provider.load_from_data(include_bytes!("style.css"));
```

Comment:
For me `include_bytes!()` didn't recompile when changing the file. If that's the case, it might result in a bit of a debugging nightmare.

Reply:
I do remember you saying that, but it does for me and I can't find anything regarding this problem on the web.